smtk/facebook.py

```diff
@@ -18,6 +17,39 @@ def __init__(self, auth=None):
 
     def on_comment(self):
         """Called when comment is found"""
+        comments = db['comment']
```

Comment:
In order to keep all storage/processing logic out of the core API's, I think the `on_*` events should be blank in the base class.  The proposed way to implement collect-social like functionality would be to create a new class that inherits and overwrites on comment with the code contained here similar to what is shown in the fourchan class in the docs or `GetFriendsLogger` class in `commands/twitter/get_friends.py`